### PR TITLE
docstring for function accept_or_reject added

### PR DESCRIPTION
**Pull request by SZL-tcc**

### CODE REVIEW

File: day_1.py [hunk 1]
```diff
@@ -84,9 +84,27 @@ def calculate_total_pair_energy(coordinates, box_length, cutoff2):
 
     return e_total
 
-def accept_or_reject(delta_e, beta):
-    # This function accepts or reject a move given the
-    # energy difference and system temperature
+def accept_or_reject(
```
Comment (b-butler) [inline day_1.py]: One argument should be on the first line and the others should be directly beneath that with the parenthesis directly after the final argument.

File: day_1.py [hunk 2]
```diff
@@ -84,9 +84,27 @@ def calculate_total_pair_energy(coordinates, box_length, cutoff2):
...
+    delta_e,        # energy difference between current & previous step
+    beta            # = 1 / reduced_temperature
+    ):
+    '''This function accepts or reject a move given the energy difference and system temperature.
```
Comment (b-butler) [inline day_1.py]: Maybe change to specify using the Metropolis criteria.

File: day_1.py [hunk 3]
```diff
@@ -84,9 +84,27 @@ def calculate_total_pair_energy(coordinates, box_length, cutoff2):
...
+
+    ----------
+    parameters
+    ----------
+    delta_e : double
+        The energy difference between the current step and the previous step.
+    beta : double
+        The inverse of reduced temperature, 1 / reduced temperature
```
Comment (b-butler) [inline day_1.py]: change to 1/T for clarity.

Comment (b-butler) [reply day_1.py]: and succinctness

File: day_1.py [hunk 4]
```diff
@@ -84,9 +84,27 @@ def calculate_total_pair_energy(coordinates, box_length, cutoff2):
...
     return e_total
 
-def accept_or_reject(delta_e, beta):
-    # This function accepts or reject a move given the
-    # energy difference and system temperature
+def accept_or_reject(
+    delta_e,        # energy difference between current & previous step
+    beta            # = 1 / T
+    ):
+    '''Accept or reject an given move basing on the Metropolis Criteria.
```
Comment (b-butler) [inline day_1.py]: a not an

File: day_1.py [hunk 5]
```diff
@@ -84,9 +84,27 @@ def calculate_total_pair_energy(coordinates, box_length, cutoff2):
...
+
+    ----------
+    parameters
```
Comment (b-butler) [inline day_1.py]: Parameters should be capitalized.

File: day_1.py [hunk 6]
```diff
@@ -84,9 +84,27 @@ def calculate_total_pair_energy(coordinates, box_length, cutoff2):
...
+    ----------
+    delta_e : double
+        The energy difference between the current step and the previous step.
+    beta : double
+        The inverse of reduced temperature, 1 / T
+
+
+    ------
+    return
```
Comment (b-butler) [inline day_1.py]: Return should be capitalized.

File: day_1.py (hunk 3 above)
Comment (SZL-tcc) [reply day_1.py]: changed

**Pull request merged**